pactman/verifier/result.py

```diff
@@ -14,6 +15,9 @@ def start(self, message):
     def end(self):
         pass
 
+    def warn(self, message):
```

Comment:
Should `Result` be a ABC ?

Reply:
Yeah, probably. I would do this in a later patch though.